charmcraft/templates/init-kubernetes/pyproject.toml.j2

```diff
@@ -1,3 +1,37 @@
+[project]
+name = "{{ name }}"
+version = "0.0.1"
+requires-python = ">=3.10"
+
+# Dependencies of the charm code in src/
+dependencies = [
+    "ops >=2.22,<3",
+]
+
+[dependency-groups]
+# Dependencies of Charmcraft libraries (from their PYDEPS)
+charmlib-pydeps = []
+# Dependencies of code in tests/unit/
+unit = [
+    "ops[testing]",
+    "pytest",
+]
+# Dependencies of code in tests/integration/
+integration = [
+    "pytest",
+    "pytest-operator",
+]
+# Dependencies needed to actually run the tests
+run-static = [
+    "pyright",
+]
+run-unit = [
```

Comment:
I don't understand why we need these to be separate -- what about just a single `unit` and `integration` (and `static`) group? Is it important or useful that they're separate?